lighthouse-core/scripts/lantern/evaluate-results.js

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env node
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-disable no-console */
+
+const path = require('path');
+
+const GOOD_ABSOLUTE_THRESHOLD = 0.2;
+const OK_ABSOLUTE_THRESHOLD = 0.5;
+
+const GOOD_RANK_THRESHOLD = 0.1;
+
+if (!process.argv[2]) throw new Error('Usage $0 <computed summary file>');
+
+const COMPUTATIONS_PATH = path.resolve(process.cwd(), process.argv[2]);
+/** @type {{sites: LanternSiteDefinition[]}} */
+const expectations = require(COMPUTATIONS_PATH);
+
+const entries = expectations.sites.filter(site => site.lantern);
+
+if (!entries.length) {
+  throw new Error('No lantern metrics available, did you run run-all-expectations.js');
+}
+
+/** @type {LanternSiteDefinition[]} */
+const totalGood = [];
+/** @type {LanternSiteDefinition[]} */
+const totalOk = [];
+/** @type {LanternSiteDefinition[]} */
+const totalBad = [];
+
+/**
+ * @param {string} metric
```

Comment:
`keyof whatever` might let you skip some of the ts-ignores below

Reply:
didn't remove ts-ignores but helps elsewhere 👍